modules/fastdds/3.1.1/MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+module(
+    name = "fastdds",
+    version = "3.1.1",
+    compatibility_level = 1,
```

Comment:
Does this version contain major breaking changes that should bump the compatibility level of this module?

See https://bazel.build/external/module#compatibility_level

Reply:
Version 2.14.x is incompatible with version 3.x.x. Please refer to https://github.com/eProsima/Fast-DDS/issues/5088

Reply:
Yes, will update